base/linalg/rowvector.jl

```diff
@@ -165,6 +165,7 @@ end
 
 # inner product -> dot product specializations
 @inline *{T<:Real}(rowvec::RowVector{T}, vec::AbstractVector{T}) = dot(parent(rowvec), vec)
+@inline *{T<:Real}(rowvec::ConjRowVector{T}, vec::AbstractVector{T}) = dot(rowvec', vec)
```

Comment:
Good question. @andyferris, why the definition beginning on lines 171/172 rather than `dot`? Thanks!